Package.swift

```diff
@@ -0,0 +1,48 @@
+// swift-tools-version:5.3
+// The swift-tools-version declares the minimum version of Swift required to build this package.
+
+import PackageDescription
+
+//#error("Automatic package resolution disabled.")
+let package = Package(
+    name: "MapboxNavigation",
+    defaultLocalization: "en",
+    platforms: [.iOS(.v10)],
+    products: [
+        // Products define the executables and libraries a package produces, and make them visible to other packages.
+        .library(
+            name: "MapboxCoreNavigation",
+            targets: [
+                "CMapboxCoreNavigation",
+                "MapboxCoreNavigation",
+            ]),
+    ],
+    dependencies: [
+        // Dependencies declare other packages that this package depends on.
+        // .package(url: /* package url */, from: "1.0.0"),
+        .package(name: "MapboxAccounts", url: "https://github.com/mapbox/mapbox-accounts-ios/", .branch("1ec5-v2.3.1")),
+//        .package(name: "MapboxDirections", url: "https://github.com/mapbox/mapbox-directions-swift/", from: "1.2.0-alpha.3"),
+//        .package(name: "MapboxMobileEvents", url: "https://github.com/mapbox/mapbox-events-ios/", from: "0.10.6"),
+//        .package(name: "MapboxNavigationNative", url: "https://github.com/mapbox/mapbox-navigation-native-ios/", from: "28.1.0"),
```

Comment:
Uncommenting any of these dependencies tends to cause Xcode to crash:

```
Crashed Thread:        10  Dispatch queue: -[IDEExecutionEnvironment initWithWorkspaceArena:] (QOS: UNSPECIFIED)

Exception Type:        EXC_BAD_INSTRUCTION (SIGILL)
Exception Codes:       0x0000000000000001, 0x0000000000000000
Exception Note:        EXC_CORPSE_NOTIFY

Termination Signal:    Illegal instruction: 4
Termination Reason:    Namespace SIGNAL, Code 0x4
Terminating Process:   exc handler [53655]

Application Specific Information:
ProductBuildVersion: 12C33
Fatal error: Duplicate values for key: 'MapboxDirections': file /AppleInternal/BuildRoot/Library/Caches/com.apple.xbs/Sources/swiftlang/swiftlang-1103.8.25.8/swift/stdlib/public/core/NativeDictionary.swift, line 747
```

Reply:
When it doesn’t crash, I get this error message instead:

> the Package.resolved file is most likely severely out-of-date and is preventing correct resolution; delete the resolved file and try again

But deleting Package.resolved, .swiftpm, the project’s entire DerivedData including SourcePackages, and the module cache doesn’t resolve the error.

Reply:
When it doesn’t crash or give that error, sometimes I get this error message instead:

> Showing All Messages
repository '/path/to/Library/Developer/Xcode/DerivedData/mapbox-navigation-ios-alubdfsnhjnshqajmpyxkgtmrvbf/SourcePackages/checkouts' has uncommited changes

In fact, that directory is a clean checkout of mapbox-accounts-ios@dbfa4bbd23d31e5d9cf6c68037dde91af1f30dd7 according to Git.

Reply:
The crash still occurs in Xcode 11.4 (which is using the same version of Swift anyhow) when pulling in any two dependencies:

```
Application Specific Information:
ProductBuildVersion: 12D4e
Fatal error: Duplicate values for key: 'MapboxMobileEvents': file /AppleInternal/BuildRoot/Library/Caches/com.apple.xbs/Sources/swiftlang/swiftlang-1103.8.25.8/swift/stdlib/public/core/NativeDictionary.swift, line 747
```

Reply:
Per [this StackOverflow post](https://stackoverflow.com/a/63079910/4585461), changing the URLs to .git URLs fixes all these issues. Xcode seems to understand a URL to a repository’s homepage, but not always.